app/views/basic_search/_form.html.erb

```diff
@@ -1,21 +1,118 @@
+          <label for="advanced-citation" class="field-label">Citation</label>
+          <input type="text" class="field field-text wide" id="advanced-citation" name="citation" placeholder="Citation" value=<%= params[:citation] %>>
+        </div>
+
+        <div class="field-wrap">
+          <label for="advanced-contributors" class="field-label">Contributors</label>
+          <input type="text" class="field field-text wide" id="advanced-contributors" name="contributors" placeholder="Contributors" value=<%= params[:contributors] %>>
+        </div>
+
+        <div class="field-wrap">
+          <label for="advanced-fundingInformation" class="field-label">Funding information</label>
+          <input type="text" class="field field-text wide" id="advanced-fundingInformation" name="Funding Information" placeholder="fundingInformation" value=<%= params[:fundingInformation] %>>
+        </div>
+
+        <div class="field-wrap">
+          <label for="advanced-identifiers" class="field-label">Identifiers</label>
+          <input type="text" class="field field-text wide" id="advanced-identifiers" name="identifiers" placeholder="Identifiers" value=<%= params[:identifiers] %>>
+        </div>
+
+        <div class="field-wrap">
+          <label for="advanced-locations" class="field-label">Locations</label>
+          <input type="text" class="field field-text wide" id="advanced-locations" name="locations" placeholder="Locations" value=<%= params[:locations] %>>
+        </div>
+
+        <div class="field-wrap">
+          <label for="advanced-subjects" class="field-label">Subjects</label>
+          <input type="text" class="field field-text wide" id="advanced-subjects" name="subjects" placeholder="Subjects" value=<%= params[:subjects] %>>
+        </div>
+
+        <div class="field-wrap">
+          <label for="advanced-title" class="field-label">Title</label>
+          <input type="text" class="field field-text wide" id="advanced-title" name="title" placeholder="Title" value=<%= params[:title] %>>
+        </div>
+
+        <div class="field-wrap">
+          <label for="advanced-source" class="field-label">Source</label>
+          <select id="advanced-source" class="field field-select" name="source">
+            <option></option>
+            <option <%= "selected" if params[:source] == "Abdul Latif Jameel Poverty Action Lab Dataverse"%>>Abdul Latif Jameel Poverty Action Lab Dataverse</option>
+            <option <%= "selected" if params[:source] == "Zenodo"%>>Zenodo</option>
+            <option>ArchivesSpace@MIT (no data yet)</option>
+            <option>DSpace@MIT (no data yet)</option>
+          </select>
+          NOTE: to allow selecting multiple sources we need to change the way we handle this in TIMDEX to treat source as an
+          "OR" search not "AND"
+        </div>
+      </div>
+    </details>
   </div>
 </form>
 
-<p><%= link_to("Advanced search", advanced_search_path) %>
+<script type="text/javascript">
```

Comment:
I'd like to see this inline javascript moved to a new file under `app/javascripts`, but that may be a fair bit of a refactor. If you'd like to hold off on that, please open a ticket to consider a refactor on this.